### DISCUSSION TITLE
Coexist with other auth methods?

### DISCUSSION THREAD

**Discussion by nallejacobsson** [ActiveLogin/ActiveLogin.Authentication#335] [Q&A] [answered]
I'm building an identity server (based on OpenIddict) with support for BankID authentication and stumbled upon this interesting project. However, I also need to support other authentication methods, like good old username/password. Any pointers regarding how ActiveLogin.Authentication can co-exist with other authentication methods in the same application? I guess my goal would be to give the user a choice between the available methods.

Thanks!

**Comment by PeterOrneholm** [comment, marked as answer]
Glad you found us. Active Login registers the BankID Authentication handler as a standard ASP.NET Authentication handler so shouldn't be any issues of registering multiple of them side by side:

These topics should be covered in the [Microsoft Docs for authentication](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/?view=aspnetcore-6.0).

**Reply by nallejacobsson** [reply]
Thanks, I'll have a look at the MS docs and give it a try 👍

**Reply by PeterOrneholm** [reply]
You are welcome!

And of cours,e if you need any assistance, we are happy to help as consultants. Things like this is what we do :)
https://www.activesolution.se/kunderbjudande/erbjudanden/saker-inloggning/

**Reply by nallejacobsson** [reply]
Our project is commercial, but there is no big budget. Would it be possible to buy a just a couple of hours of consulting?

**Reply by PeterOrneholm** [reply]
It sure is. We have an offering for that exact purpose. It's the support package, read more under "Active Login Support" at:
https://www.activesolution.se/kunderbjudande/erbjudanden/saker-inloggning/

(And yes that table is TERRIBLE. I'll make sure to fix the layout of it soon...)
